lib/Filter.php

```diff
@@ -72,6 +72,7 @@ public static function formatHumanReadableSize($size)
     /**
      * fixed time string comparison operation to prevent timing attacks
      * https://crackstation.net/hashing-security.htm?=rd#slowequals
+     * can be replaced with hash_equals() after we drop PHP 5.5 support
```

Comment:
Oughm… good point.

Actually, constant time comparison is important. I guess it's actually off-topic for this PR, but why can't we just check whether `hash_equals` exists and use it. Otherwise just fallback to that custom `slowEquals`?